doc/raid_on_backend.rst

```diff
@@ -45,6 +48,52 @@ Attaching volume
 There's a `ansible configuration`_ for this, and `list of volumes`_.
 
 
+Adding more space
+-----------------
+
+1. Create two ``gp3`` volumes in EC2 of the same size and type, tag them with
+   ``FedoraGroup: copr``, ``CoprInstance: production``, ``CoprPurpose:
+   infrastructure``.  Attach them to a freshly started temporary instance (we
+   don't want to overload I/O with the initial RAID sync on production
+   backend).  Make sure the instance type has enough EBS throughput to perform
+   the initial sync quickly enough.
+
+2. Always partition the disks with a single partition on them, otherwise kernel
+   might have troubles to auto-assemble the disk arrays::
+
+        cfdisk /dev/nvmeXn1
+        cfdisk /dev/nvmeYn1
+
+3. Create the ``raid1`` array on both the new **partitions**::
+
+        $ mdadm --create --name=raid-be-03 --verbose /dev/mdXYZ --level=1 --raid-devices=2 /dev/nvmeXn1p1 /dev/nvmeYn1p1
+
+   Wait till the new array is synchronized (may take hours or days).  Check the
```

Comment:
I don't understand two things at this point.

- In the first step, why do we create two new volumes instead of just one?
- We are waiting for the new array to be synchronized. Is it an empty array without any data, right? Maybe you could change the sentence to "Wait till the new empty array" so it is absolutely clear. It really takes days to sync empty array?

Reply:
> why do we create two new volumes instead of just one?

We do?  X and Y.

> Is it an empty array without any data, right? 

Yes.

> It really takes days to sync empty array?

It depends on the volume size and I/O speed (of the volumes, and actual instance type, which needs to handle twice as much during sync) .  Note we create 16T volumes.  https://raid.wiki.kernel.org/index.php/Initial_Array_Creation

Reply:
Thank you for the review btw! :) I tried to update the things you touched. ptal